src/Service/Cache.php

```diff
@@ -208,14 +208,15 @@ public static function clearCache( array $tags = [] ) {
 	 * Calls clearCache using WP Cron.
 	 * Why? ClearCache can be resource intensive on larger instances and should be offloaded.
 	 *
-	 * @param array $tags
+	 * @param array $tags to clear cache for
 	 *
 	 * @return void
 	 */
 	public static function scheduleClearCache( array $tags = [] ) {
-		$event = wp_schedule_single_event( time(), self::$clearCacheHook, [ $tags ] );
+		$event = wp_schedule_single_event( time(), self::$clearCacheHook, [ $tags ], true );
```

Comment:
@hansmorb An dieser Stelle wurde wp_schedule_single_event mit dem Parameter für Rückgabe von WP_Error als default false aufgerufen. Deswegen ist die Rückgabe-Signatur von `wp_schedule_single_event(...)` gleich `bool` und `is_wp_error(...)` wird immer `false` ausgewertet. Das hat PHPStan erkannt.

Reply:
Das war in der Theorie ein Bug, für die Fälle, in denen wp_schedule_single_event fehlschlägt und unser Code damit die Cache Bereinigung nicht erzwingt. Ich weiß aktuell aber nicht wie das auftreten kann.

Reply:
Ah, das habe ich übersehen. Ja das ist doch eine sinnvolle Änderung!